packages/design-system/src/components/DateField/DateInput.jsx

```diff
@@ -51,71 +51,51 @@ export class DateInput extends React.PureComponent {
     }, 20);
   }
 
-  render() {
+  renderField(type) {
     const sharedTextFieldProps = {
       className: 'ds-l-col--auto',
       labelClassName: 'ds-c-datefield__label',
+      disabled: this.props.disabled,
       inversed: this.props.inversed,
       onBlur: (this.props.onBlur || this.props.onComponentBlur) && this.handleBlur,
       onChange: this.props.onChange && this.handleChange,
       numeric: true,
     };
 
+    // Add additional id to `aria-describedBy` with bottom placed errors
```

Comment:
I generalized rendering of the day, month and year fields

Reply:
Clever reuse of `classNames` 👍

Reply:
Nice refactoring.